README.md

```diff
@@ -25,13 +25,17 @@ Simple API of the npm generator:
 wc-generate --name [wc name]
 ```
 
+### Minimum Node version
+
+The WC-Generator requires a minimum install of Node.js `14.0.0`.
```

Comment:
@blackfalcon is 14 really required? I've been able to run fine with 12.

Reply:
Sorry, I was testing things and the easiest way was to use Github versus getting all up in my nvms. 

The minimum is 11. Recommended is 14.